scripts/us_census/pep/population_estimate_by_race/preprocess.py

```diff
@@ -0,0 +1,936 @@
+    df['0'] = df[0]
+    df['1'] = df[1]
+    df['2'] = df[2]
+    df['3'] = df[3]
+    df['4'] = df[4]
+    df['5'] = df[5]
+    df['6'] = df[6]
+    df['Year'] = df['level_0'].str[-4:]
+    df = df.drop(['level_0', 0, 1, 2, 3, 4, 5, 6], axis=1)
+    df.columns = df.columns.str.replace('NAME', 'Geographic Area')
+    df.columns = df.columns.str.replace('0', 'Total')
+    df.columns = df.columns.str.replace('1', 'White Alone')
+    df.columns = df.columns.str.replace('2', 'Black or African American Alone')
+    df.columns = df.columns.str.replace('3',\
+        'American Indian or Alaska Native Alone')
+    df.columns = df.columns.str.replace('4', 'Asian Alone')
+    df.columns = df.columns.str.replace('5',\
+        'Native Hawaiian and Other Pacific Islander Alone')
+    df.columns = df.columns.str.replace('6', 'Two or more Races')
+    print(df.columns)
+    return df
+
+
+def _clean_xlsx_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a xls file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame) : DataFrame of xls dataset
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for xls dataset.
+    """
+    df = df.drop(['Census', 'Estimates Base'], axis=1)
+    df = df.drop([1], axis=0)
+    df.drop(df.index[7:], inplace=True)
+    #print(df.columns)
+    df['Unnamed: 0'] = df['Unnamed: 0'].str.replace(".", "")
+    df['Geographic Area'] = 'United States'
+
+    # it groups the df as per columns provided
+    # performs the provided functions on the data
+
+
+    df = df.groupby(['Geographic Area','Unnamed: 0']).sum()\
+        .transpose().stack(0).reset_index()
+    df.columns = df.columns.str.replace('level_0', 'Year')
+    df.columns = df.columns.str.replace('White', 'White Alone')
+    df.columns = df.columns.str.replace('TOTAL POPULATION', 'Total')
+    df.columns = df.columns.str.replace('Black or African American', \
+        'Black or African American Alone')
+    df.columns = df.columns.str.replace(
+        'Native Hawaiian and Other Pacific Islander'\
+        , 'Native Hawaiian and Other Pacific Islander Alone')
+    df.columns = df.columns.str.replace(
+        'American Indian and Alaska Native'\
+        , 'American Indian or Alaska Native Alone')
+    df.columns = df.columns.str.replace('Asian', 'Asian Alone')
+
+    df['Total'] = pd.to_numeric(df['Total'])
+    # print(df)
+    return df
+
+
+def _clean_county_70_xls_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a xls file format.
+    Also, Performs transformations on the data.
+
+    Arguments:
+        df (DataFrame) : DataFrame of xls dataset
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for xls dataset.
+    """
+    df['Total People']=df[1]+df[2]+df[3]+df[4]+df[5]+df[6]+df[7]\
+        +df[8]+df[9]+df[10]+df[11]+df[12]+df[13]+df[14]+df[15]+\
+        df[16]+df[17]+df[18]
```

Comment:
for i in range(1,18):
    df['Total People'] += df[i]

Reply:
created loop for this above statement